docs/config/README.md

```diff
@@ -93,15 +94,15 @@ For example:
 
 - `(@$region)` will show nothing if the variable `region` is `None`, otherwise `@` followed by the value of region.
 - `(some text)` will always show nothing since there are no variables wrapped in the braces.
-- When `$all` is a shortcut for `\[$a$b\] `, `($all)` will show nothing only if `$a` and `$b` are both `None`.
+- When `$all` is a shortcut for `\[$a$b\]`, `($all)` will show nothing only if `$a` and `$b` are both `None`.
   This works the same as `(\[$a$b\] )`.
```

Comment:
IDK if these changes are introduced by a markdown formatter, but the space is left intentionally here.  If the space is removed above, the corresponding one here should also be removed.

Reply:
It looks like they were. 😥
I'll go through and fix them up.